ports/servoshell/headed_window.rs

```diff
@@ -522,33 +517,32 @@ impl WindowPortsMethods for Window {
     }
 }
 
+fn winit_size_to_euclid_size<T>(size: PhysicalSize<T>) -> Size2D<T, DevicePixel> {
+    Size2D::new(size.width, size.height)
+}
+
+fn winit_position_to_euclid_point<T>(position: PhysicalPosition<T>) -> Point2D<T, DevicePixel> {
+    Point2D::new(position.x, position.y)
+}
```

Comment:
This would be a little bit more idiomatic as an implementation of `From` or `Into`. At the moment, I cannot recall the current rules about implementing these traits on foreign types.

Reply:
both types are foreign, so sadly we can’t do that in servoshell. (we could however do that in euclid, if we added an optional winit feature or something)